### PR TITLE
Halls1

### PR DESCRIPTION
**Pull request by nagarajanvisha**

### CODE REVIEW

File: medical/doctors.html [hunk 1]
```diff
@@ -184,13 +261,158 @@ <h3>Address</h3>
...
+            </dialog>
+          </li>
         </ul>
       </div>
     </aside>
     <p>Visit our <a href="https://www.facebook.com/groups/kovaipudurr" target="_blank">facebook group</a> page</p>
   </main>
   <footer>
-    <p>&copy;2021 kovaipudur</p>
+    <p>&copy; 2021 covaipudur.in | <a href="/disclaimer.html">Disclaimer</a></p>
```
Comment (rajakvk) [inline medical/doctors.html:415]: This looks good.

**Review by rajakvk — CHANGES_REQUESTED** [review]
This looks good.

File: medical/doctors.html (hunk 1 above)
Comment (nagarajanvisha) [reply medical/doctors.html:415]: Fine. Please merge.

**Review by nagarajanvisha — COMMENTED** [review]
Fine. Please Merge.

**Pull request merged**